spec/models/billing_contract_spec.rb

```diff
@@ -1,21 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe BillingContract, type: :model do
-  let!(:billing_contract) { FactoryBot.build(:billing_contract) }
+  let!(:billing_contract) { FactoryBot.create(:billing_contract) }
 
   it 'has a valid factory' do
     expect(billing_contract).to be_valid
   end
 
-  describe 'validations' do
+  context 'when valid associations' do
```

Comment:
better to move it in describe block, and call it just `describe 'associations'....`
can associations be invalid?